flake.nix

```diff
@@ -11,6 +11,8 @@
       inputs.nixpkgs.follows = "nixpkgs";
     };
 
+    nixGL.url = "github:guibou/nixGL";
```

Comment:
```
nixGL = {
  url = "github:guibou/nixGL";
  inputs.nixpkgs.follows = "nixpkgs";
};
```
You don't wanna copy nixpkgs too much

Reply:
Will it create a copy of nixpkgs if I'm not explicit in what it should follow?

Reply:
Yup, any dependencies of nixgl would be duplicated all the way down to stdenv (doesn't have to be a lot, but could be multiple glibc, python, bash yadayada